bfinst

```diff
@@ -449,7 +449,7 @@ if [ -z "${skip_boot_update}" ]; then
     # MUST be executed after 'install_grub', otherwise the newly
     # created boot option would be either cleaned up or unset
     # from default option.
-    bfrec
+    bfrec --policy dual
```

Comment:
Please notify Vlad about this new flag. He needs to add this flag in his install script

Reply:
Done